test/int/atlas_streams_test.go

```diff
@@ -55,7 +55,7 @@ YJZC5C0=
 -----END CERTIFICATE-----
 `
 
-	BeforeEach(func() {
+	BeforeEach(func(ctx context.Context) {
```

Comment:
Well TIL, this is way too magic... but it is good for this case!

Reply:
Why do you think is magic?

Reply:
Because **BeforeEach** accepts almost _whatever_ as an argument